payments/models.py

```diff
@@ -731,11 +737,11 @@ def set_status(self, new_status: OrderStatus, comment: str = None) -> None:
             OrderStatus.PAID: (OrderStatus.REFUNDED,),
             OrderStatus.PAID_MANUALLY: (OrderStatus.REFUNDED,),
             OrderStatus.ERROR: (
```

Comment:
Let's add also `DRAFTED` here, following the comment on `leases/services/invoice.py`